### DISCUSSION TITLE
wiki/getting-started/controller-support/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#664] [Wiki]
# wiki/getting-started/controller-support/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/getting-started/controller-support/

**Comment by Tobiytytyt** [comment]
There is a mod available for 1.12 and earlier
its called Joypad Splitscreen Mod 
its available at https://github.com/ljsimin/MinecraftJoypadSplitscreenMod
and http://retro-hack.blogspot.com/p/minecraft-joypad-mod.html
(not mine just mentioning also sorry for bad English not mi main language)

**Comment by EchoEllet** [comment]
Controlify mod now supports NeoForge (from 1.21 to 1.21.8). See the [supported versions](https://modrinth.com/mod/controlify/versions?l=neoforge).
